Fix mentioning of non-existing page on Marts page (#7905)
dbt-labs/docs.getdbt.com: website/docs/best-practices/how-we-structure/4-marts.md

---

:::info

> Our guidance here diverges if you use the <Constant name="semantic_layer" />. In a project without the <Constant name="semantic_layer" /> we recommend you denormalize heavily, per the best practices below. On the other hand, if you're using the <Constant name="semantic_layer" />, we want to stay as normalized as possible to allow MetricFlow the most flexibility. You can find more information at the bottom of this page: [The dbt Semantic Layer and marts](/docs/best-practices/how-we-structure#the-dbt-semantic-layer-and-marts).

Reviewer: ```suggestion
Our guidance here diverges if you use the <Constant name="semantic_layer" />. In a project without the <Constant name="semantic_layer" /> we recommend you denormalize heavily, per the best practices below. On the other hand, if you're using the <Constant name="semantic_layer" />, we want to stay as normalized as possible to allow MetricFlow the most flexibility. See [The dbt Semantic Layer and marts](#the-dbt-semantic-layer-and-marts) for more information.
```
